### ISSUE TITLE
Czech accented characters not working in search

### ISSUE THREAD

**Issue by Baegus** [sublimehq/sublime_merge#1208]
**Version info**

 - OS: Windows 10
 - Build: 2057

**Description**

Searching for a commit message only returns commits if there are no accented characters (such as ě, š, č, ř ...) in the search query.

**Steps to reproduce**

1. Create a Czech commit message, such as "Příliš žluťoučký kůň"
2. Search for that same string.
3. Nothing gets returned.

**Expected behavior**

The commit should be returned.

**Comment by dpjohnst** [comment]
Hi @Baegus,

Thank you for reporting this issue!

So far I've been unable to reproduce this issue, and it seems to be working correctly on my machine with the provided steps.

![Match](https://user-images.githubusercontent.com/18252447/125007180-08fcac00-e0a3-11eb-891b-74f8753ad179.png)

When you have a spare moment, could you share a screenshot of the search query you're using in Sublime Merge?

Cheers,
\- Dylan

**Comment by Baegus** [comment]
@dpjohnst You're right, it does work! The only problem is that these characters are apparently case sensitive (my commit was named "Češi" and i searched for the lowercase version of that ("češi").
